modules/plugins/randompictures.go

```diff
@@ -144,10 +144,11 @@ func (rp *RandomPictures) Init(session *discordgo.Session) {
 			cursor, err := rethink.Table("randompictures_sources").Run(helpers.GetDB())
 			helpers.Relax(err)
 			err = cursor.All(&rpSources)
-			helpers.Relax(err)
-			if err != nil && err != rethink.ErrEmptyResult {
-				helpers.Relax(err)
+			if err == rethink.ErrEmptyResult {
+				raven.CaptureError(fmt.Errorf("%#v", err), map[string]string{})
+				continue
```

Comment:
I added 'continue' here.
Please check this is right.

Reply:
continue is great, but same as above, EmptyResult => continue, other erros => helpers.Relax